http/src/main/resources/reference.conf

```diff
@@ -168,29 +168,30 @@ gke {
     location = "us-central1-a"
   }
   defaultNodepool {
-    machineType = "g1-small",
-    numNodes = 1,
+    machineType = "g1-small"
+    numNodes = 1
     autoscalingEnabled = false
   }
   galaxyNodepool {
-    machineType = "n2-standard-8",
-    numNodes = 1,
-    autoscalingEnabled = true,
-    autoscalingConfig = {
-       autoscalingMin = 0,
+    machineType = "n2-standard-8"
+    numNodes = 1
+    autoscalingEnabled = true
+    autoscalingConfig {
+       autoscalingMin = 0
        autoscalingMax = 1
     }
   }
-  app {
-    releaseName = "release1",
-    namespaceNameSuffix = "namespace",
+  galaxyApp {
```

Comment:
Renamed `app` to `galaxyApp` since it really contains Galaxy-specific configs (and we may need to add more Galaxy configs in the future).